### ISSUE TITLE
Optimize and simplify the NLog.config for Nethermind.Runner

### ISSUE THREAD

**Issue by snakefoot** [NethermindEth/nethermind#1773]
You can remove the `AutoFlushWrapper`:

https://github.com/NethermindEth/nethermind/blob/c17cafdb57aea27a0809d53766bd99b30dddc1d7/src/Nethermind/Nethermind.Runner/NLog.config#L29

And instead use the NLog ColoredConsole option [autoFlush="true"](https://github.com/NLog/NLog/wiki/ColoredConsole-target)

When using `<targets async="true">` :

https://github.com/NethermindEth/nethermind/blob/c17cafdb57aea27a0809d53766bd99b30dddc1d7/src/Nethermind/Nethermind.Runner/NLog.config#L12

Then it automatically enables use of AsyncWrapper for all targets, so there is no need to add them again.

https://github.com/NethermindEth/nethermind/blob/340abf74723f49b83afca4fa2e241f8302c0f5e1/src/Nethermind/Nethermind.Runner/NLog.config#L13-L20

https://github.com/NethermindEth/nethermind/blob/c17cafdb57aea27a0809d53766bd99b30dddc1d7/src/Nethermind/Nethermind.Runner/NLog.config#L30-L36

Remember to update all `<rules>` when removing the explicit-async and flush (Change from `file-async` to `file` and from `auto-colored-console-async` to `colored-console`)

https://github.com/NethermindEth/nethermind/blob/c17cafdb57aea27a0809d53766bd99b30dddc1d7/src/Nethermind/Nethermind.Runner/NLog.config#L71-L72

**Comment by tkstanczak** [comment]
cool, thank you, this is exactly what we have been discussing recently as we had some sever performance issues with logging to file and Seq

@snakefoot if we remove the async wrappers how can we provide the overflow action and other parameters? (other than currently used defaults)

https://github.com/NethermindEth/nethermind/pull/1775

**Comment by snakefoot** [comment]
When using `<targets async="true">` then it is just like doing this:

```xml
<targets>
    <default-wrapper xsi:type="AsyncWrapper" overflowAction="Discard" timeToSleepBetweenBatches="1" batchSize="200" />

    ...
</targets>
```

You are free to use `<default-wrapper>` instead of using `async="true"`.

See also: https://github.com/NLog/NLog/wiki/performance

**Comment by snakefoot** [comment]
Resolved by #1775
